esFill.js

```diff
@@ -0,0 +1,8 @@
+const exports = {
```

Comment:
What does this file do?

Reply:
Yep, sorry, another fragment, I made a branch off of an older commit.